pkg/platform/runtime/buildscript/buildscript.go

```diff
@@ -79,7 +80,7 @@ func NewScript(data []byte) (*Script, error) {
 
 	script, err := parser.ParseBytes(constants.BuildScriptFileName, data)
 	if err != nil {
-		return nil, errs.Wrap(err, "Could not parse build script")
+		return nil, locale.WrapInputError(err, "err_parse_buildscript_bytes", "Could not parse build script: {{.V0}}", errs.JoinMessage(err))
```

Comment:
Should we assume an input error here?

Reply:
Hmm good question. The parser does use our parsing logic right? So if an error happened it could be due to something we did and not the user?

Reply:
It depends. 99% of the time this will be a user error because they introduced a syntax error in the buildscript. We have unit tests that do round-trip testing, so if we're improperly writing the buildscript, then we should catch it before we get to this point. Technically it's possible we're writing a bad build script though if we have a complicated and/or unexpected buildexpression we're translating. It's pretty unlikely in my opinion. I'm okay with this being an input error.

Reply:
Correct, we define the language schema and the parser does it's work based on that. So we could get errors if:
1. We made a mistake in the language schema
2. Some unexpected error from the parsing library
3. A buildscript is actual malformed

We would likely want to report to rollbar for the first two types of errors and the third is the one we disregard. It's will be hard to filter those out though.

I was able to find a `participle.Error` interface that I'm now using in the error check. This should allow us to be more confident that they are parsing errors. I'm wrapping the participle errors as input and the others as regular local errors for now.

Reply:
Awesome, thanks guys! I'll defer to your better judgment.